### ISSUE TITLE
"lint whole folder" task ignores eslint.rules.customizations

### ISSUE THREAD

**Issue by twastvedt** [microsoft/vscode-eslint#1304]
In my settings file I have:
```
"eslint.rules.customizations": [{ "rule": "*", "severity": "downgrade" }]
```

When I run "eslint: lint whole folder", however, eslint errors show up as errors in the Problems panel. If I click on one of those errors, then the offending file is opened, the regular eslint process runs that file, and all issues that were reporting as errors are switched (because of the customizations) to reporting as warnings. Shouldn't the lint whole folder task look at customizations and show those issues as warnings in the first place?

**Comment by dbaeumer** [comment]
Might be very hard to address since the lint task simply executes the code in a terminal without parsing and interpreting the output.

**Comment by dbaeumer** [comment]
The feature has not gained any traction in the community. I therefore close the issue. Still a big Thank You to you for taking the time to create this issue! To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!
